module/VuFind/src/VuFind/Auth/Shibboleth/SingleIdPConfigurationLoading.php

```diff
@@ -0,0 +1,35 @@
+<?php
+namespace VuFind\Auth\Shibboleth;
+
+class SingleIdPConfigurationLoading implements ConfigurationLoadingInterface
```

Comment:
I would suggest replacing "Loading" with "Loader" in the class and interface names; most of our classes are named for what they do, and I think "Loader" would read more consistently.